tests/queries/0_stateless/02366_kql_native_interval_format.sql

```diff
@@ -0,0 +1,22 @@
+select 'default format';
+select toIntervalNanosecond(99);
+select toIntervalNanosecond(100);
+select toIntervalMillisecond(1);
+select toIntervalSecond(42);
+select toIntervalMinute(66);
+select toIntervalHour(66);
+select toIntervalDay(5);
+select toIntervalWeek(1);
+select toIntervalWeek(2);
+
+select 'kql format';
+set interval_format = 'kql';
+select toIntervalNanosecond(99);
```

Comment:
We allow mixed intervals as well like 1 hour 30 minutes. You can get it as a sum of intervals. 

I propose to rewrite tests to jinja style and mix all the interval types to see it works correctly

Reply:
@qoega I think that mixing types is a perfectly reasonable proposal. Would you perhaps have an example of what this jinja style entails?

Reply:
It's all done, thanks for the tips!